app/ui/browser-modern/views/browser/location/index.jsx

```diff
@@ -68,12 +68,12 @@ class Location extends Component {
 
   handleInputKeyDown = e => {
     if (e.key === 'Enter') {
-      this.handleAutocompletionPick({ uri: e.target.value });
+      this.handleAutocompletionPick({ url: e.target.value });
```

Comment:
Unrelated, but this also needs to be fixed -- it should be `{ data: { url: e.target.value } }`, if I read the function below correctly.